### PR TITLE
[update] license year

### PR DESCRIPTION
**Pull request by josephlin55555**
This pull request updates the license year to cover until the current year (2018).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2008, Batiste Bieler
+Copyright (c) 2008-2018, Batiste Bieler
```
Comment (czpython) [inline LICENSE]: Please use 2008-present to avoid the update every year.

Comment (josephlin55555) [reply LICENSE]: Updated

**Comment by josephlin55555** [comment]
Updated license to state `2008-present`

**Comment by josephlin55555** [comment]
Updated license to state `2008-present` @czpython

**Comment by czpython** [comment]
Thanks!

**Pull request merged**